logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/client/MarkMLSE2EIdEnableChangeAsNotifiedUseCaseTest.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.kalium.logic.feature.client
+
+import com.wire.kalium.logic.configuration.UserConfigRepository
+import com.wire.kalium.logic.feature.user.MarkMLSE2EIdEnableChangeAsNotifiedUseCase
+import com.wire.kalium.logic.feature.user.MarkMLSE2EIdEnableChangeAsNotifiedUseCaseImpl
+import com.wire.kalium.logic.functional.Either
+import io.mockative.Mock
+import io.mockative.any
+import io.mockative.eq
+import io.mockative.given
+import io.mockative.mock
+import io.mockative.once
+import io.mockative.verify
+import kotlinx.coroutines.test.runTest
+import kotlin.test.Test
+
+class MarkMLSE2EIdEnableChangeAsNotifiedUseCaseTest {
```

Comment:
```suggestion
class MarkMLSE2EIEnableChangeAsNotifiedUseCaseTest {
```

TBH I can't understand the name. Mark MLS E2EI Enable (?) Change (?)